docs/index.rst

```diff
@@ -1,103 +1,82 @@
-    # IMPORTANT: Use only whitespace, but NO TAB indentation in this YAML file!
-    ################################################################################
-    chip_dict:
-      my_H3K27ac_sample:
-        control: my_Input_sample
-        broad: False
-      my_H3K27me3_sample:
-        control: my_Input_sample
-        broad: True
 
+Running your own analysis
+-----------------------------
 
-Genome configuration file
-----------------------------
+For a detail introduction to setting up snakePipes from scratch, please visit :doc:`/content/setting_up.rst`
 
-Any organism can be supported in the workflows by adding a genome configuration file `my_organism.yaml` in the
-following style to the `snakemake_workflows` directory:
+For each organism of interest, snakePipes requires fasta files, genome indexes and annotation files.
+Paths to these files are specified in the organism/<name>.yaml files. After installation, the location
+of these files could be revealed by the following command:
 
 .. code:: bash
 
-    $ cat snakemake_workflows/shared/organisms/hs37d5.yaml
+    snakePipes info
+
+You could either modify the existing files (add your own paths), or add a new file there. See more detail in
+:doc:`/content/running_snakePipes.rst`
 
-.. parsed-literal::
+snakePipes could either be executed locally, or on any snakemake-supported cluster infrastructure. See details
+for setting up the cluster command in :doc:`/content/running_snakePipes.rst`
 
-	genome_size: 2900338458
-	genome_fasta: "/SOMEPATH/hs37d5_ensembl/genome_fasta/genome.fa"
-	genome_index: "/SOMEPATH/hs37d5_ensembl/genome_fasta/genome.fa.fai"
-	genome_2bit: "/SOMEPATH/hs37d5_ensembl/genome_fasta/genome.2bit"
-	bowtie2_index: "/SOMEPATH/hs37d5_ensembl/BowtieIndex/genome"
-	hisat2_index: "/SOMEPATH/hs37d5_ensembl/HISAT2Index/genome"
-	known_splicesites: "/SOMEPATH/hs37d5_ensembl/gencode/release_19/HISAT2/splice_sites.txt"
-	star_index: "/SOMEPATH/hs37d5_ensembl/STARIndex/"
-	genes_bed: "/SOMEPATH/hs37d5_ensembl/gencode/release_19/genes.bed"
-	genes_gtf: "/SOMEPATH/hs37d5_ensembl/gencode/release_19/genes.gtf"
-	blacklist_bed: "/SOMEPATH/hs37d5_ensembl/ENCODE/hs37d5_extended_Encode-blacklist.bed"
+Citation
+---------
 
+If you adopt/run snakePipes for your analysis, cite it as follows :
+
+    snakePipes: flexible and scalable NGS analysis pipelines built using snakemake.
+    The MPI-IE Bioinformatics Facility. http://doi.org/10.5281/zenodo.1146540
+
+
+.. image:: content/images/logo_mpi-ie.jpg
+
+This tool suite is developed by the `Bioinformatics Unit <http://www.ie-freiburg.mpg.de/bioinformaticsfac>`_
+at the `Max Planck Institute for Immunobiology and Epigenetics <http://www.ie-freiburg.mpg.de/>`_, Freiburg.
 
-.. note:: If no blacklist regions are available for your organism of interest, leave `blacklist_bed:` empty
+
+Help and Support
+-----------------
+
+For query/questions regarding snakePipes, please write to the deepTools user group : `deeptools@googlegroups.com`
```

Comment:
Let's let the mailing list die. Github and biostars are sufficient. I plan to phase out support on the deepTools mailing list at the end of the year too.